docs/introduction/directories.md

```diff
@@ -0,0 +1,19 @@
+<!--- TODO
+# Directories
+This article describes how to work with directories and how to create your own directories.
+
+## Config
+All directories that need to be created are defined in `app/config/directories.yml` and are created by symfony command `shopsys:create-directories`.
+
+## Directories type
+There are 2 types of directories that are created.
+
+### Local filesystem directories
+Directories for local filesystem are used for all directories that are used by application and do not need to be public.
+
+### Main filesystem directories
```

Comment:
Is `Main` and `Local` terms we already use? Because from the names I don't know what these do.